core/extension/gdextension_interface.h

```diff
@@ -790,16 +790,35 @@ typedef struct {
 	const char *string;
 } GDExtensionGodotVersion;
 
+typedef struct {
+	uint32_t major;
+	uint32_t minor;
+	uint32_t patch;
+	const char *status;
+	const char *string;
+} GDExtensionGodotVersion2;
```

Comment:
Should we use the opportunity to document each field with a short example value, so it's clear how they look?